### PR TITLE
feat: Add confirm password option in new registration screen

### LINKED ISSUE #474: Confirm Password Option In new registration screen
**Summary:**

There should be a `Confirm Password` Option also in the new account registration screen.

Something like this.
account

### PR DESCRIPTION
**Pull request by Rish02Sharma**
Fixes #474 

account

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Review by luckyman20 — COMMENTED** [review]
Instead of using Toast, you should use Toaster class and show a Snackbar.

**Comment by therajanmaurya** [comment]
@Rish02Sharma Thanks for sending the PR. Please fix conflicts and fix the Travis build too. Here you can find out the reports(https://github.com/openMF/self-service-app/wiki/Travis-Fail-because-of-Quality-Plugin.) of the Travis build and why It's failing

**Comment by Rish02Sharma** [comment]
@therajanmaurya Updated the branch and fixed the travis build too. :+1:

File: app/src/main/res/layout/fragment_registration.xml [hunk 1]
```diff
@@ -84,6 +84,18 @@
             android:inputType="textPassword"/>
     </android.support.design.widget.TextInputLayout>
 
+    <android.support.design.widget.TextInputLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/default_margin"
+        app:passwordToggleEnabled="true">
+        <EditText android:id="@+id/et_ConfirmPassword"
```
Comment (therajanmaurya) [inline app/src/main/res/layout/fragment_registration.xml]: @Rish02Sharma we are not following camel case in xml ids. it should be `et_confirm_password`

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -116,6 +116,8 @@
     <string name="rb_mobile">mobile</string>
     <string name="rb_email">email</string>
     <string name="verification_mode">Verification Mode</string>
+    <string name="confirmPassword">Confirm Password</string>
```
Comment (therajanmaurya) [inline app/src/main/res/values/strings.xml]: @Rish02Sharma please add the string in all strings.xml, whenever you add a new string. Here is a medium blog(https://medium.com/@therajanmaurya/add-localization-string-in-application-supported-all-languages-using-keyboard-shortcut-807e0a388672). I have written, have a look

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -116,6 +116,8 @@
...
+    <string name="error_password">Password doesnot match.</string>
```
Comment (therajanmaurya) [inline app/src/main/res/values/strings.xml]: this should be like
` <string name="error_password_not_match">Password doesnot match.</string>`

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_registration.xml (hunk 1 above)
Comment (Rish02Sharma) [reply app/src/main/res/layout/fragment_registration.xml]: Done! :+1:

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (Rish02Sharma) [reply app/src/main/res/values/strings.xml]: Done! :+1:

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (Rish02Sharma) [reply app/src/main/res/values/strings.xml]: Done! :+1:

**Review by therajanmaurya — APPROVED** [review]

**Pull request merged**